File: streaming/src/main/java/com/welfarerobotics/welfareapplication/streaming/view/SignInActivity.java
package com.welfarerobotics.welfareapplication.streaming.view;

import android.databinding.DataBindingUtil;
import com.welfarerobotics.welfareapplication.streaming.R;
import com.welfarerobotics.welfareapplication.streaming.base.BaseActivity;
import com.welfarerobotics.welfareapplication.streaming.databinding.SignInView;
import com.welfarerobotics.welfareapplication.streaming.viewmodel.SignInViewModel;

/**
 * @author : Hyunwoong
 * @when : 2019-07-03 오전 1:11
 * @homepage : https://github.com/gusdnd852
 */
public class SignInActivity extends BaseActivity<SignInView, SignInViewModel> {

    @Override protected int bindView() {
        return R.layout.sign_in_view;
    }

    @Override protected void observe() {

    }
}

File: streaming/src/main/java/com/welfarerobotics/welfareapplication/streaming/view/SignInActivity.kt
package com.welfarerobotics.welfareapplication.streaming.view

import android.databinding.DataBindingUtil
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.welfarerobotics.welfareapplication.streaming.R
import com.welfarerobotics.welfareapplication.streaming.base.BaseActivity
import com.welfarerobotics.welfareapplication.streaming.databinding.SignInViewBinding
import com.welfarerobotics.welfareapplication.streaming.viewmodel.SignInViewModel


/**
 * @author : Hyunwoong
 * @when : 2019-07-02 오후 9:50
 * @homepage : https://github.com/gusdnd852
 */
class SignInActivity : BaseActivity() {

    lateinit var view: SignInViewBinding
    lateinit var viewModel: SignInViewModel

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        view = DataBindingUtil.setContentView(this, R.layout.sign_in_view)
        viewModel = factory.create(SignInViewModel::class.java)
    }
}